cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-scaling.adoc

```diff
@@ -0,0 +1,302 @@
+----
+
+* You can also verify autoscaling from the {cluster-manager} 
++
+. In the OpenShift web console navigational menu, click *Observe > Dashboards*.
+. In the dashboard, select *Kubernetes / Compute Resources / Namespace (Pods)* and your namespace *ostoy*.
++
+image::deploy-scale-hpa-metrics.png[Select metrics]
++
+. A graph appears showing your resource usage across CPU and memory. The top graph shows recent CPU consumption per pod and the lower graph indicates memory usage. The following lists the call outs in the graph: 
+.. The load started to increase (A), 
+.. Two new pods started to spin up (B, C). 
+.. The thickness of each graph is its CPU consumption, indicating which pods handled more load. 
+.. You can also see that the load decreased (D), after which, the pods were spun back down.
++
+image::deploy-scale-metrics.png[Select metrics]
+
+=== Node Autoscaling
+
+{product-title} allows you to use link:https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.html[node autoscaling]. In this scenario, you will create a new project with a job that has a large workload that the cluster cannot handle. With autoscaling enabled, when your load is larger than you can currently handle, it should force the cluster to automatically create new nodes to handle the load.
+
+.Prerequisites
+
+* Autoscaling is enabled on your machine pools.
+
+.Procedure
+
+. Create a new project called `autoscale-ex` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project autoscale-ex
+----
+
+. Create the job by running the following command:
++
+[source,terminal]
+----
+$ oc create -f https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/yaml/job-work-queue.yaml
+----
++
+
+. Allow the pods some time to create and then, run the following command to see the pods:
++
+[source,terminal]
+----
+$ oc get pods
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                     READY   STATUS    RESTARTS   AGE
+work-queue-5x2nq-24xxn   0/1     Pending   0          10s
+work-queue-5x2nq-57zpt   0/1     Pending   0          10s
+work-queue-5x2nq-58bvs   0/1     Pending   0          10s
+work-queue-5x2nq-6c5tl   1/1     Running   0          10s
+work-queue-5x2nq-7b84p   0/1     Pending   0          10s
+work-queue-5x2nq-7hktm   0/1     Pending   0          10s
+work-queue-5x2nq-7md52   0/1     Pending   0          10s
+work-queue-5x2nq-7qgmp   0/1     Pending   0          10s
+work-queue-5x2nq-8279r   0/1     Pending   0          10s
+work-queue-5x2nq-8rkj2   0/1     Pending   0          10s
+work-queue-5x2nq-96cdl   0/1     Pending   0          10s
+work-queue-5x2nq-96tfr   0/1     Pending   0          10s
+----
+
+. Because there are many pods in a `Pending` state, this status should trigger the autoscaler to create more nodes in your machine pool. Allow more time to pass to create these worker nodes.
```

Comment:
```suggestion
. Because there are many pods in a `Pending` state, this status should trigger the autoscaler to create more nodes in your machine pool. Allow time to create these worker nodes.
```